extra_geom/detectors.py

```diff
@@ -1342,3 +1342,119 @@ def __init__(self, modules, filename='No file'):
         warnings.warn(
             "DSSC_Geometry has been renamed to DSSC_1MGeometry.", stacklevel=2
         )
+
+
+class Jungfrau_Geometry(DetectorGeometryBase):
```

Comment:
Naming nitpicks: PSI always write JUNGFRAU in block capitals, as it's technically an acronym, even if it's a really awkward one. And for the other detectors, the underscore is within the detector type name, which we don't need here.

So: `JUNGFRAUGeometry`? Or do we ignore their capitalisation and go for the more legible `JungfrauGeometry`?

Reply:
JUNGFRAUGeometry.  
Note that I got rid of the redundant "_" as well